src/ui/handler/map_event.js

```diff
@@ -90,19 +91,19 @@ export class MapEventHandler {
         this._map.fire(new MapTouchEvent(e.type, this._map, e));
     }
 
-    _firePreventable(mapEvent: MapMouseEvent | MapTouchEvent | MapWheelEvent) {
+    _firePreventable(mapEvent: MapMouseEvent | MapTouchEvent | MapWheelEvent): void | HandlerResult {
         this._map.fire(mapEvent);
         if (mapEvent.defaultPrevented) {
             // returning an object marks the handler as active and resets other handlers
-            return {};
+            return (({}: any): HandlerResult);
```

Comment:
Alternatively, we could remove the exact signifier (`{...}` instead of `{|...|}`) — then we can avoid `any` cast here. https://github.com/facebook/flow/issues/2977